recipes/festim/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "festim" %}
+{% set version = "1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/festim-{{ version }}.tar.gz
+  sha256: 7ebefae709a8981183cf7d118dceec71681464f496b74837a8e5a966b052d63c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=45
+    - wheel
+    - setuptools-scm >=7.0.5
+    - pip
+  run:
+    - python >={{ python_min }}
+    - fenics
```

Comment:
You may need to report this in the fenics feedstock:

```
fenics-dolfin 2019.1.0 has requirement pybind11==2.2.4, but you have pybind11 2.13.6.
```

Reply:
pinging @minrk @jorgensd can we fix this?

Reply:
We prefer if this can be fixed upstream. Otherwise you will force all packages that depend on this to skip pip-check.

Reply:
Something else might be holding fenics-dolfin back. The [latest build](https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Ffenics-dolfin-2019.1.0-py39he78686c_54.conda) requires pybind11==2.13.6

Reply:
It is unconstrained in the recipe: https://github.com/conda-forge/fenics-feedstock/blob/997d6afd946dfa52f86ee2737314d99cdae5383f/recipe/meta.yaml#L112

Reply:
@minrk is this something that will be an issue down the line when trying to use FESTIM from conda or can we merge this PR?